src/Propel/Generator/Manager/MigrationManager.php

```diff
@@ -439,9 +464,11 @@ public function findMigrationClassNameSuffix(int $timestamp): string
         $path = $this->getWorkingDirectory();
         if ($path && is_dir($path)) {
             $files = scandir($path);
```

Comment:
```suggestion
            $files = scandir($path) ?: [];
```
saves you the if case